tasks/html-css-responsive-layouts.md

```diff
@@ -1,3 +1,9 @@
+[![MIT Licensed][icon-mit]][license]
+[![Ideas and useful links][icon-ideas]][ideas]
+[![Awesome][icon-awesome]][awesome]
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+[![Gitter][icon-chat]][chat]
```

Comment:
Markdown reference links require _reference keys_ that looks like following:

```markdown
Somewhere in the file we show [link][ref]

<!-- somewhere else in the file -->
[ref]: http://www.example.com
```

This approach allows us to have less text when we typing, or reference links when finished the paragraph. Also, it's really useful when you want to use the same link few times per page.

Just copy first 8 ref-links from the README's footer.